test/blockchaintest/blockchaintest_runner.cpp

```diff
@@ -29,6 +29,7 @@ struct TransitionResult
     state::BloomFilter bloom;
     int64_t blob_gas_left;
     TestState block_state;
+    bool system_contracts_valid = true;
```

Comment:
We don't check this flag yet and tests pass... We should take a look how we check blockchain test results.

Reply:
I added a comment to another line explaining what needs to be done https://github.com/ethereum/evmone/pull/1199#discussion_r2044113080